sorc/checkout.sh

```diff
@@ -161,7 +161,7 @@ if [[ ${checkout_gsi} == "YES" ]]; then
 fi
 
 if [[ ${checkout_gdas} == "YES" ]]; then
-  checkout "gdas.cd" "https://github.com/NOAA-EMC/GDASApp.git" "843d3a9"; errs=$((errs + $?))
+  checkout "gdas.cd" "https://github.com/NOAA-EMC/GDASApp.git" "e27973d"; errs=$((errs + $?))
```

Comment:
Note:  We may need to update the GDASApp hash before this PR is merged into g-w develop.